code/noRAM/app/src/main/java/com/example/noram/model/PushNotificationService.java

```diff
@@ -77,14 +79,16 @@ public void onMessageReceived(RemoteMessage remoteMessage) {
      * @param title the title of the notification
      * @param data the data of the notification
      * @param event the event to send the notification to
+     * @param isOrganizer a boolean to check if we wish to only send the notification to the organizer
      */
-    public void sendNotification(String title, String data, Event event) {
+    public void sendNotification(String title, String data, Event event, Boolean isOrganizer) {
```

Comment:
"isOrganizer" is a bit of a confusing variable name here imo, but its not a _huge_ deal

Reply:
anything u think is better?

Reply:
I changed it